src/service/cia301/co_csdo.c

```diff
@@ -62,7 +56,7 @@ CO_ERR COCSdoRequestDownloadFull(CO_CSDO *csdo,
                              blockTransfer_t* block);
 
 // cbt TODO: Update UploadFull function to take blockTransfer_t instead of a block boolean
-CO_ERR COCSdoRequestUploadFull(CO_CSDO *csdo,
+static CO_ERR COCSdoRequestUploadFull(CO_CSDO *csdo,
```

Comment:
obligatory const request: does csdo change? does the timeout or buffer pointer?

Reply:
Won't do - out of scope for these changes is my excuse and sticking to it - would like it if someone did this - put a ticket in manta backlog but unlikely to be prioritized ...

Reply:
Copy that ticket is MANT-948